main.tf

```
@@ -21,7 +21,10 @@ resource "aws_s3_bucket" "athena-workspace" {
     }
   }
 
-  tags = var.tags
+  tags = merge(
+    var.tags,
+    lookup(var.resource_specific_tags, "bucket", {})
```

does it maybe make sense to always use the resource name like this?
```suggestion
    lookup(var.resource_specific_tags, "s3_bucket", {})
```

---

@fmasuhr and also this?

```hcl
lookup(var.resource_specific_tags, "athena_workgroup", {})
```

---

Right then we change that one as well :)